docs/pull_request_template.md

```diff
@@ -20,7 +20,7 @@ Fixes # (PUT ISSUE NUMBER AFTER #)
   - If you added config, describe how we use this tool in the config’s comment.
   - If you added something to the project’s architecture, describe it in the project’s `README.md`.
   - Try to focus on “why?”, not “how?”.
-- [ ] If you added a new dependency, check our [requirements](../README.md#dependencies).
+- [ ] If you added a new dependency, check our [requirements](/hplush/slowreader/blob/main/README.md#dependencies).
```

Comment:
Try to open this link from https://github.com/pshergie/slowreader/blob/fix-pr-body-checklist-link/docs/pull_request_template.md

Reply:
The case you described is not common as the file is intended to be interacted with through the pull request page but I fixed it though. Let's hope github doesn't change its URL soon